Formula/astyle.rb

```diff
@@ -3,7 +3,7 @@ class Astyle < Formula
   homepage "http://astyle.sourceforge.net/"
   url "https://downloads.sourceforge.net/project/astyle/astyle/astyle%202.05.1/astyle_2.05.1_macosx.tar.gz"
   sha256 "de66da286dee2b9de1dc1c05092cbf5368c0889f25d1e2ee8b51766aff8e4baf"
-  head "svn://svn.code.sf.net/p/astyle/code/trunk/AStyle"
+  head "https://svn.code.sf.net/p/astyle/code/trunk/AStyle"
```

Comment:
These SVN changes will probably need to be reverted I'm afraid. We've historically used either `svn://` or `http://` because the system `svn` can't handle automatic SSL/TLS connections and we're not interested in mandating people install Homebrew's SVN just to do SVN checkouts. Example:
```
Error validating server certificate for 'https://svn.code.sf.net:443':
 - The certificate is not issued by a trusted authority. Use the
   fingerprint to validate the certificate manually!
Certificate information:
 - Hostname: *.code.sf.net
 - Valid: from Wed, 06 Apr 2016 00:00:00 GMT until Mon, 05 Jun 2017 23:59:59 GMT
 - Issuer: GeoTrust Inc., US
 - Fingerprint: 9c:63:de:5c:24:c4:86:ed:d2:53:26:e1:26:6b:52:0d:dd:b0:95:a3
(R)eject, accept (t)emporarily or accept (p)ermanently?
```